.github/workflows/ci.yml

```diff
@@ -0,0 +1,63 @@
+name: ci
+
+on: [push]
+
+env:
+  GO_VERSION: 1.24.1
+  RETENTION-DAYS: 1
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Setup Go ${{ env.GO_VERSION }}
+        uses: actions/setup-go@v5
+        with:
+          go-version: ${{ env.GO_VERSION }}
+      - name: build
+        run: cd ./cli/alphabill && go build -o ../../build/alphabill
+      - name: upload build artifact
```

Comment:
not sure the binary is needed, especially with retention of 1 day. there's docker image and potentially a "release" with binaries